dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -768,7 +815,7 @@ public void constantField_notUpdatedCache_whenChanged() throws Exception{
             contentletAPI.publish(pageEnglishVersion, systemUser, false);
             addAnonymousPermissions(pageEnglishVersion);
 
-            MultiTree multiTree = new MultiTree(pageEnglishVersion.getIdentifier(), containerId,
+            MultiTree multiTree = new MultiTree(pageEnglishVersion.getIdentifier(), container.getIdentifier(),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'multiTree' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5025402)